### ISSUE TITLE
Header Parameter

### ISSUE THREAD

**Issue by MichaelMay81** [Zaid-Ajaj/Hawaii#35]
I have a range paramter that belongs in the Header.
Hawaii recognizes that and creates:
``` fsharp
async {
    let requestParts =
        [ ...
          if interval.IsSome then
              RequestPart.query ("interval", interval.Value)
          if range.IsSome then
              RequestPart.header ("range", range.Value)
          if name.IsSome then
              RequestPart.query ("name", name.Value)
          ... ]

    let! (status, content) =
        OpenApiHttp.getAsync url "/repositories/{repository}/timeseries" headers requestParts
```
but in sendAsync only the parameter extraHeaders is used for Headers.
So ranges work if I define them as part headers in the class instance of my client, but not as part of my json attributes.

Suggested solution:
Filter Header definitions from parts in sendAsync and merge them with extraHeaders.

**Comment by Zaid-Ajaj** [comment]
Thanks @MichaelMay81 for opening the issue! I am a bit low on time these days to debug and fix these types of issues. Any chance you could send a PR for it? I will find time tho to review, merge and publish 😉

**Comment by MichaelMay81** [comment]
Just trying to get my problem and planned solution to you early, so I am not working on a useless pull request :)
But it is done now. Tell me if it is useful :)
PullRequest #36

**Comment by Zaid-Ajaj** [comment]
Resolved as of v0.60 🚀
